src/twisted/conch/ssh/session.py

```diff
@@ -91,6 +91,25 @@ def request_pty_req(self, data):
         else:
             return 1
 
+    def request_env(self, data):
+        """
+        Process a request to pass an environment variable.
+
+        @type data: L{bytes}
```

Comment:
Please add `@param` and `@returns` as well, per the [coding standard](https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html).

Reply:
Done.